tw-NEDa-frontend/src/app/app.module.ts

```diff
@@ -31,10 +40,14 @@ import { Md2DateUtil } from 'md2-datepicker/datepicker/dateUtil';
     MaterialModule.forRoot(),
     FlexLayoutModule,
     NgxDatatableModule,
-    Md2DatepickerModule
+    Md2DatepickerModule,
+    AgmCoreModule.forRoot({
+      apiKey: 'AIzaSyBCUrIt3JLF2DlRX6pgDA9ulLZl41uHnTY'
```

Comment:
Can you add a comment to specify what this is doing to share the knowledge across the team?